modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -3551,6 +3621,46 @@ private ObjectAction _createObjectAction(String objectActionName) {
 		return objectAction;
 	}
 
+	private int _getObjectEntryVersionsCountByObjectDefinition(
+		long objectDefinitionId) {
+
+		return _objectEntryVersionLocalService.dslQueryCount(
+			DSLQueryFactoryUtil.countDistinct(
+				ObjectEntryVersionTable.INSTANCE.objectEntryVersionId
+			).from(
+				ObjectEntryVersionTable.INSTANCE
+			).innerJoinON(
+				ObjectEntryTable.INSTANCE,
+				ObjectEntryTable.INSTANCE.objectEntryId.eq(
+					ObjectEntryVersionTable.INSTANCE.objectEntryId)
+			).where(
+				ObjectEntryTable.INSTANCE.objectDefinitionId.eq(
+					objectDefinitionId)
+			));
+	}
+
+	private long _getSharingEntriesCountByObjectDefinition(
```

Comment:
I'm in doubt, but I think the pattern is to inline this logic.

Reply:
I am using it more than one time.